doc/configuration.md

```diff
@@ -78,3 +81,7 @@ This method allows you to set the default endpoint to use. This means, that when
 ## 7: Chain
 
 All methods return `this`, allowing you to chain the calls.
+
+## 8: Set default base URL
+
+All endpoints registered after this call will have the passed in URL prepended by default, rather than the current host. 
```

Comment:
isn't it doing 
All endpoints registered without configure function or specific host use the default URL, rather than the current host. 
or?